packages/circuit-ui/components/LoadingButton/LoadingButton.stories.tsx

```diff
@@ -38,7 +38,12 @@ export const Base = (args: LoadingButtonProps) => {
   };
 
   return (
-    <LoadingButton isLoading={loading} onClick={handleClick} {...args}>
+    <LoadingButton
```

Comment:
This entire file should be deleted. We don't want to encourage people to continue using this component.